apps/web-giddh/webpack.partial.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
             'isCordova': JSON.stringify(false),
             'errlyticsNeeded': JSON.stringify(false),
             'errlyticsKey': JSON.stringify(''),
-            'AppUrl': JSON.stringify('https://beta.giddh.com/'),
```

Comment:
is it required to remove the trailing slash?

Reply:
Updated